analyzers/its/regression-test.ps1

```diff
@@ -118,7 +116,7 @@ function Initialize-ActualFolder() {
     }
 
     # this copies no files if ruleId is not set, and all but ending with ruleId if set
-    Copy-FolderRecursively -From .\expected -To .\actual -Exclude "*${ruleId}.json"
+    Copy-FolderRecursively -From .\expected -To .\actual -Exclude "${ruleId}*.json"
```

Comment:
nasty little remaining bug from the previous PR, since the `ruleId` got moved to the _beginning_ of the name.